connect.js

```diff
@@ -25,15 +26,18 @@ function connect({
     onSuccess = arguments[1].onSuccess || isRequired("onSuccess callback");
     onLoad = arguments[1].onLoad || anonFunc;
     rest = {};
+    onEvent = arguments[1].onEvent || anonFunc;
```

Comment:
this block handles a deprecated Mono connect constructor, so won't support new features